sormas-e2e-tests/src/test/resources/features/sanity/web/Contacts.feature

```diff
@@ -18,3 +18,14 @@ Feature: Contacts end to end tests
       Then I open the last created contact
       Then I delete the contact
       And I check that number of displayed contact results is 0
+
+  @EditContact @issue=5634 @testTicket=5406
```

Comment:
Please remove the `@testTicket=5406` there is no logic for this

Reply:
done. the logic is that it could provide unique keys per scenario.
this would be helpful later if you want to check in jenkins a serialization of failed tests.